airgun/entities/subscription.py

```diff
@@ -145,6 +145,48 @@ def delete(self, entity_name):
         self._wait_for_process_to_finish(
                 'Delete Upstream Subscription', has_manifest=True)
 
+    def read_header(self):
+        """Read the head columns under Subscription"""
+        view = self.navigate_to(self, 'All')
+        return view.table.headers
+
+    def reset_scc(self):
```

Comment:
that should be placed on test layer whether in test module helper or if you have only one test case for that - then in test itself

Reply:
I'm not too sure what you mean here.

Reply:
I mean that framework should not contain such helpers. If you need select any checkbox, so do it in the test directly. If you need to select all of them (I am not sure why you need it), so just do that still in the test

Reply:
When you say "if you need to select any checkboxes, so do it in the test directly" do you mean write that in ui_airgun/test_subscription.py or where?

I only did the select all or not select all because I'm ensuring that all or no headers show up correctly based on those 2 filtering.